lib/Dialect/ESI/ESIOps.cpp

```diff
@@ -639,6 +639,58 @@ void ESIPureModuleOp::setHWModuleType(hw::ModuleType type) {
   emitError("No ports for port types");
 }
 
+//===----------------------------------------------------------------------===//
+// Manifest ops.
+//===----------------------------------------------------------------------===//
+
+StringRef ServiceImplRecordOp::getManifestClass() { return "service"; }
```

Comment:
can these raw strings be moved to some named constants?

Reply:
They're not (as of now) used in any other part of the code -- they simply appear in the manifest. So I don't think that warrants a named constant.